packages/client/src/features/tools/change-bounds/change-bounds-tool-feedback.ts

```diff
@@ -280,10 +280,12 @@ export class FeedbackMoveMouseListener extends MouseListener implements Disposab
             if (moveAction) {
                 result.push(moveAction);
             }
+            const feebackCleanup: Action[] = [];
```

Comment:
```suggestion
            const feedbackCleanup: Action[] = [];
```
Although getting a fee back also sound good 😉

Reply:
Ha! Good catch!